pkg/sql/opt/memo/testdata/stats/inverted-geo

```diff
@@ -534,3 +534,25 @@ project
       │              └── fd: (3)-->(5)
       └── filters
            └── st_intersects('010200000002000000000000000000E03F000000000000E03F666666666666E63F666666666666E63F', g:2) [type=bool, outer=(2), immutable, constraints=(/2: (/NULL - ])]
+
+# Tests for multi-column inverted spatial indexes.
+
+exec-ddl
+CREATE TABLE m (
+  k INT PRIMARY KEY,
+  a INT,
+  g GEOMETRY,
+  INVERTED INDEX (a, g)
+)
+----
+
+# The stats builder should not panic during a SELECT on a table with a
+# multi-column inverted index.
+opt
+SELECT * FROM m
```

Comment:
I put this here because the stats builder required a minor tweak to not panic when performing any SELECT on a table with a multi-col inverted index. But it's not really testing statistics. LMK if there's a better place for this test.